src/main/java/org/fenixedu/academic/domain/evaluation/config/MarkSheetSettings.java

```diff
@@ -13,19 +15,9 @@ protected MarkSheetSettings() {
         setRoot(Bennu.getInstance());
     }
 
-    public static void init() {
-        if (getInstance() == null) {
-            makeInstance();
-        }
-    }
-
-    public static MarkSheetSettings getInstance() {
-        return Bennu.getInstance().getMarkSheetSettings();
-    }
-
     @Atomic
-    private static void makeInstance() {
-        new MarkSheetSettings();
+    public static MarkSheetSettings getInstance() {
```

Comment:
This way all the current invocation of this method will start a write transaction, potentially degrading the system.